tests/test_stormevent.py

```diff
@@ -212,3 +215,25 @@ def test_storm_event_coops_product_within_region(florence2018):
 
     assert null_tidal_data["t"].sizes == {}
     assert east_coast_tidal_data.sizes == {"nos_id": 111, "t": 1}
+
+
+def test_status():
+    florence2018 = StormEvent("florence", 2018)
+    paine2016 = StormEvent.from_nhc_code("EP172016")
+    henri2021 = StormEvent.from_usgs_id(310)
+    ida2021 = StormEvent("ida", 2021)
+
+    assert florence2018.status == StormStatus.HISTORICAL
+    assert paine2016.status == StormStatus.HISTORICAL
+    assert henri2021.status == StormStatus.HISTORICAL
+    assert ida2021.status == StormStatus.HISTORICAL
+
+    storms = nhc_storms()
+    latest_storm_entry = storms.iloc[-1]
+    latest_storm = StormEvent.from_nhc_code(latest_storm_entry.name)
+    if pandas.isna(
+        latest_storm_entry["end_date"]
+    ) or datetime.today() - latest_storm_entry["end_date"] > timedelta(days=1):
```

Comment:
I think this time it's the reverse of the other issue above. The condition must be 
```python
if not pandas.isna(...) or diff > timedelta(days=1):
   assert latest_storm.status == StormStatus.HISTORICAL
else:
   assert latest_storm.status == StormStatus.REALTIME
```

Reply:
Actually this results in NA comparison, since if it's NA, the first part of short circuit check is False so it goes to the next. But in terms of logic this is how the check should be. Maybe we can do
```python
if pandas.isna(...) or diff < timedelta(days=1):
   assert latest_storm.status == StormStatus.REALTIME
else:
   assert latest_storm.status == StormStatus.HISTORICAL
```
in both the test and actual status check